graylog2-web-interface/src/preflight/components/ConfigurationWizard/RenewalPolicyConfiguration.tsx

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import * as React from 'react';
+import { Formik, Form, Field } from 'formik';
+import styled from 'styled-components';
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import moment from 'moment';
+
+import { Title, Space, Button, Group, NumberInput, Input } from 'preflight/components/common';
```

Comment:
In the past I created the `Input` component, which is a mantine `TextInput`. Having another look I prefer to follow the mantine structure by creating a `TextInput` and an `Input` component. I will refactor this in a follow-up PR.